internal/characters/alhaitham/plunge.go

```diff
@@ -36,6 +36,12 @@ func (c *char) LowPlungeAttack(p map[string]int) action.ActionInfo {
 		}
 	}
 
+	short := p["short"]
+	skip := 0
+	if short > 0 {
+		skip = 20
```

Comment:
why not have this param be a variable amount for folks to choose how much delay/how high the plunge is? Maybe rename this to `height` where 0 is the old behavior and we add more delay as `height` increases?

Reply:
because this is supposed to only give you a quick option between the super short variant and the shortest long variant. if you need any more frames on top you should be using the pre-action function/waits.

Reply:
The range of frames between the two variants above is what I would consider to be "impossible" to do in-game. Either you do the short variant or you're already at the long variant frame count at the very least.